app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -308,7 +308,7 @@ public void onCreate(Bundle savedInstanceState) {
             return;
         }
 
-        contacts = Contacts.getInstance(MessageCompose.this);
+        contacts = DI.get(Contacts.class);
```

Comment:
Please move to where the other dependencies are looked up near the top of the class.